src/platform/virt.rs

```diff
@@ -36,6 +37,10 @@ static VIRT_CLINT: VirtClint = VirtClint::new(&CLINT_MUTEX);
 pub struct VirtPlatform {}
 
 impl Platform for VirtPlatform {
+    fn name() -> &'static str {
+        "Virtual"
```

Comment:
Maybe "QEMU virt", it's how I have seen it called elsewhere (and `virt` is the official machine name in QEMU, we invoke it with `--machine virt`).